api/config/config.go

```diff
@@ -24,6 +24,8 @@ type Config struct {
 		Name                   string `yaml:"name"`
 		UsersCollectionName    string `yaml:"usersCollectionName"`
 		ProjectsCollectionName string `yaml:"projectsCollectionName"`
+		PrivKeysCollectionName string `yaml:"privKeysCollectionName"`
```

Comment:
move to keystore config

Reply:
actually lets just change the `database` config name to `mongo` since it really is mongo specific config, but we can do this in a subsequent PR.